src/javascript/app/pages/trade/charts/highchart.js

```diff
@@ -488,20 +488,16 @@ const Highchart = (() => {
     // calculate where to display the maximum value of the x-axis of the chart for line chart
     const getMaxHistory = (history_times) => {
         let end = end_time;
-        if (sell_spot_time && (sell_time || sell_spot_time) < end_time) {
-            end = sell_spot_time;
+        if (sell_time && sell_time < end_time) {
+            end = sell_time;
         } else if (exit_tick_time) {
             end = exit_tick_time;
         }
 
         const history_times_length = history_times.length;
         if (is_settleable || is_sold) {
-            for (let i = history_times_length - 1; i >= 0; i--) {
-                if (parseInt(history_times[i]) === end) {
-                    max_point = parseInt(history_times[i === history_times_length - 1 ? i : i + 1]);
-                    break;
-                }
-            }
```

Comment:
since we use `sell_time` now, sometimes it is between ticks, which means `parseInt(history_times[i]) === end` is never true